stream/src/main/scala/libretto/stream/scaletto/BinarySearchTree.scala

```diff
@@ -428,26 +428,25 @@ class BinarySearchTree[DSL <: Scaletto, CLib <: CoreLib[DSL], SLib <: ScalettoLi
   }
 
   private object Absorptive {
-    implicit def fromTransportive[F[_]](using F: Transportive[F]): Absorptive[F] =
-      new Absorptive[F] {
-        override def lift[A, B](f: A -⚬ B): F[A] -⚬ F[B] =
-          F.lift(f)
+    given [F[_]](using F: Transportive[F]): Absorptive[F] with {
```

Comment:
Is this clear enough without name? I think so, but not 100% sure.

Reply:
I think so. Also, it's private stuff, so it's visibility/usage is quite localized.